src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
+    &-button {
```

Comment:
```suggestion
    &--button {
```

Reply:
Same as https://github.com/sparkletown/sparkle/pull/2336#discussion_r714654480

Reply:
fixed b230401

Reply:
@alexey-petrov but why "fix" something that wasn't broken? :D @yurii-lubynets look what you did now :D